api/grpcserver/grpcserver_test.go

```diff
@@ -2268,7 +2269,7 @@ func TestEventsReceived(t *testing.T) {
 	time.Sleep(time.Millisecond * 50)
 
 	lg := zaptest.NewLogger(t)
-	svm := vm.New(statesql.InMemory(), vm.WithLogger(lg))
+	svm := vm.New(statesql.InMemory(), vmsdk.WithLogger(lg))
```

Comment:
Then revert here and in all the other places:
```suggestion
	svm := vm.New(statesql.InMemory(), vm.WithLogger(lg))
```